docs/docsite/rst/dev_guide/testing/sanity/compile.rst

```diff
@@ -12,9 +12,9 @@ All Python source files must successfully compile using all supported Python ver
 
 Control node code, including plugins in Ansible Collections, must support the following Python versions:
 
-- 3.11
-- 3.10
-- 3.9
+- 3.14
+- 3.13
+- 3.12
```

Comment:
Looks like this also needs updating in the stable branches... 3.9-3.11 is somewhat outdated.